care/facility/api/serializers/patient.py

```diff
@@ -453,9 +454,18 @@ def create(self, validated_data):
 
     def save(self, **kwargs):
         self.instance.facility = self.validated_data["facility"]
-        PatientConsultation.objects.filter(
+        consultation = PatientConsultation.objects.filter(
             patient=self.instance, discharge_date__isnull=True
-        ).update(discharge_date=localtime(now()))
+        ).first()
+
+        consultation.discharge_date = localtime(now())
```

Comment:
```suggestion
        consultation.discharge_date = now()
```

we should pass timezone to db wherever possible